conda_gitenv/tests/integration/test_resolve.py

```diff
@@ -8,18 +8,48 @@
 
 
 class Test_full_build(unittest.TestCase):
-    def test_basic_env(self):
+    def check_basic_env(self):
         repo = setup_samples.basic_repo()
```

Comment:
@corinnebosley I think my travis carnage stems from here, in that each test should use a separate repo ... not a similarly named singleton ... just a thought.